easybuild/easyblocks/o/opencv.py

```diff
@@ -90,9 +90,6 @@ def prepare_step(self, *args, **kwargs):
     def configure_step(self):
         """Custom configuration procedure for OpenCV."""
 
-        if 'CMAKE_BUILD_TYPE' not in self.cfg['configopts']:
-            self.cfg.update('configopts', '-DCMAKE_BUILD_TYPE=Release')
```

Comment:
The idea was to NOT use `-DCMAKE_BUILD_TYPE=` at all but rather specify `build_type`. I don't want to encourage users to do that. And configopts overwrites build_type so it doesn't hurt.

Reply:
But we do have a bunch of easyconfigs now that specify `-DCMAKE_BUILD_TYPE=Release` (and there are more out there that we don't control).

For those, we shouldn't specify `-DCMAKE_BUILD_TYPE` again, even if the one in `configopts` wins, because that'll lead to confusion (why is it there twice, which one wins, etc.).

We can do a cleanup sweep in easyconfigs to use `build_type = 'Release'` rather than use `-DCMAKE_BUILD_TYPE=Release` (or just leave it to the default), if you think that's worth it.

Reply:
> But we do have a bunch of easyconfigs now that specify -DCMAKE_BUILD_TYPE=Release

Most (all?) should be fixed by this (in particular https://github.com/easybuilders/easybuild-easyblocks/pull/1929/commits/7289d42993f9dfa150ec3442ce7efb08e93cdb6d)

FTR: I actually consider this "confusing" as good. It shows that someone is using the old approach and should switch to the new one to make the output less confusing (at least some incentitive)

Reply:
Solution: Show a warning/info that build_type is overwritten